agent/config.go

```diff
@@ -15,6 +17,13 @@ func FixupCheckType(raw interface{}) error {
 		return nil
 	}
 
```

Comment:
having a comment or link to the issue here would be useful in the future for anyone looking here. +Also maybe a cautionary note to not add more mappings like this and to use the :json annotation for new fields.

Reply:
I'll add that.

Reply:
done